dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -19,8 +19,7 @@
 import static com.google.cloud.tools.opensource.classpath.TestHelper.COORDINATES;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-
-import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
```

Comment:
Insert an empty line between static imports and non-static imports.

Reply:
done